MDISforLinux/DRIVERS/BBIS/SMB2BB/DRIVER/NATIVE/bb_smb2.c

```diff
@@ -142,10 +142,14 @@ typedef struct smb_access_struct {
 /*
  *  SMBus registering/access functions differ a bit throughout 2.4 to 2.6.x.
  */
-static int oss_smb2_probe(struct i2c_client* client, const struct i2c_device_id *id);
-static int oss_smb2_probe(struct i2c_client *client, const struct i2c_device_id *id);
+static int  oss_smb2_probe(struct i2c_client* client, const struct i2c_device_id *id);
+static int  oss_smb2_probe(struct i2c_client *client, const struct i2c_device_id *id);
```

Comment:
oss_smb2_probe function prototype is duplicated (lines 145 and 146). Additionally, the unique change I saw is adding an extra space between the returning type (int) and the function name.

Reply:
How did I get to this? I'll double check